### PR TITLE
Tutorial for how to use the jacobian of mjx.step

### PR DESCRIPTION
**Pull request by Andrew-Luo1**
A notebook showing how to use vanilla policy gradients to accelerate policy learning, thanks to MJX's differentiability. Demonstrations are given for imitation learning and locomotion, for a quadruped. Requires [Brax PR 476](https://github.com/google/brax/pull/476).

**Supporting changes**
1. Add an Anymal C model, cleaned for faster simulation on MJX similarly to MJX [Barkour](https://github.com/google-deepmind/mujoco/tree/main/mjx/mujoco/mjx/test_data/barkour_v0)
2. Add an image for the notebook

### CODE REVIEW

**Comment by erikfrey** [comment]
Exciting!  How many changes did you make to anymal?  Can you try this:

- Make a PR to mujoco menagerie with `anymal_c_mjx.xml` and `scene_mjx.xml` with your anymal modifications
- Somewhere at the top of your colab, do:

```sh
!git clone https://github.com/google-deepmind/mujoco_menagerie.git
```

Then you should be able to remove the anymal assets from this PR.  Let me know if that works!

**Comment by Andrew-Luo1** [comment]
Hi Erik, I've proposed [Menagerie PR 50](https://github.com/google-deepmind/mujoco_menagerie/pull/50) with the Anymal C updates, and have documented the modifications in the readme. I've updated this notebook as well.

The formatting of this notebook isn't quite the same as the current mujoco notebooks. In the case that this PR goes through, of course feel free to modify it as you see fit: adding the Mujoco banner, adding a colab instance, etc!

And let me know if there's anything I can do to bring the notebook more up to standards.

File: mjx/pyproject.toml [hunk 1]
```diff
@@ -44,4 +44,4 @@ mjx-viewer = "mujoco.mjx.viewer:main"
 Homepage = "https://github.com/google-deepmind/mujoco/tree/main/mjx"
 Documentation = "https://mujoco.readthedocs.io/en/3.1.5"
 Repository = "https://github.com/google-deepmind/mujoco/tree/main/mjx"
-Changelog = "https://mujoco.readthedocs.io/en/3.1.5/changelog.html"
+Changelog = "https://mujoco.readthedocs.io/en/3.1.5/changelog.html"
```
Comment (erikfrey) [inline mjx/pyproject.toml:47]: I suspect this is an unneeded change - can you revert?  Otherwise, let me know what this is for.

**Review by erikfrey — CHANGES_REQUESTED** [review]
OK, I see your change is incorporated into menagerie.  Looking great!  One last nit and we will merge.

**Pull request merged**